capellambse/diagram/_styleclass.py

```diff
@@ -30,6 +30,8 @@ def get_styleclass(obj: model.ModelObject) -> str:
 
 
 def _default(obj: model.ModelObject) -> str:
+    if type(obj).__name__ == "GenericElement":
+        return obj.xtype.split(":")[-1]  # type: ignore[attr-defined]
```

Comment:
This should use a proper `isinstance(obj, model.GenericElement)` check instead, which would also avoid the `type: ignore`.